docs/architecture/adr/adr-006-in-cluster-db.md

```diff
@@ -0,0 +1,170 @@
+6-8 months.
+
+The main points we kept debating was:
+
+- Can we operate an in-cluster database on par or better than Azure?
+- Are we able to deal with problems that might arise on the databases with out
+Azures Technical Support?
+- If we self host the databse, are we then able to optimize the performance?
+- Can we guarantee a better or on par uptime to Azure's managed database if we
+selfhost the database?
+- Would we get better support if we moved to the suggested managed database?
+- Would the performance be better on the database suggested by Azure?
+- Would downtime be minimized if we moved to the sugggested manged database?
+
+We have continually, and to a worsening degree, seen even more downtime since
+summer. We have been cut-off from having insight into the logs of the database
+due to an extremely steep price on server logs, which we tried enabling during
+the summer. This resulted in a more than doubled operations cost, which the
+operations budget in now way can bear.
+We have on serveral occasions found the database to have been misconfigured, to
+such a degree that it has caused unreasonable downtime.
+
+The expertise and technical support of Azure has been a factor in favor of
+migrating to Azure manged MySql database. As mentioned before we have already
+seen Azure neglience to setup database configured to correctly to the specs
+readily available from MariaDB's documentation on operating a database.
+We have on multiple occasions experienced that the database has crashed due to
+e.g. too many connection, too many slow queries, which should be the exact thing
+the database should have been configured to handle in a reasonable way.
+The reasonable behaviour in the afforementioned cases would have been, "too
+many connections" and "Ressource busy", which should have resulted in end-user
+experiences such as 500 errors and not project/nation wide downtime.
+
+Conversations with the Azure Technical Support is seeped in effective solution
+conversations. This is due to:
+
+- Language barriers as the Azure support is handled by people whos English
+proficiency is wildly varying.
+- The tiered support which bars us from talking to the actual technical staff.
+This results in a knowledge relay bottleneck, often resulting in
+lost-in-translation situations.
+- Tranparency issues stemming from somewhere inside of the technical support,
+where information is withheld in such a way that it builds mistrust to Azure.
+
+Lastly we have experienced Azure Technical Support taking unsanctioned actions
+on the database, resulting in project wide downtime on serveral occasions. The
+first time we exprienced this was summer 2024, where they restarted the databse
+while doing some configuration. That last time was january 2025, where they
+restarted the database multiple times trying to fix a bad database recovery.
+
+Deranged has dealt with managed databases in the past, and has never experienced
+a level of service as bad as this.
```

Comment:
Too much. Too subjective. Tone should be more objective and sober.

yes, we're frustrated - but that is irrelevant to the decision.